src/prommis/solvent_extraction/solvent_extraction.py

```diff
@@ -156,3 +156,23 @@ def mass_transfer_term(b, t, s, k, l, m):
             self.mscontactor.stream_component_interactions,
             rule=mass_transfer_term,
         )
+
+        self.clean_sx_pe_tank_cap = Var(
```

Comment:
I think all of this needs to be removed. Some of these already exist in the model elsewhere (number and volume of the tanks), some are not even part of the SX unit operation (e.g. pumps), and the rest are things that you do not have correlations for and should be added only as part of costing the flowsheet (e.g., mixer power).

Reply:
Okay I will remove them

Reply:
I have removed them